<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSvgCollection</class>
 <widget class="QWidget" name="WdgSvgCollection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>288</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KisSqueezedComboBox" name="cmbCollections"/>
     </item>
     <item>
      <widget class="QToolButton" name="vectorPresetsConfigureButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listCollection"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>KisSqueezedComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
